### ISSUE TITLE
Typo in autosummary template "module.rst" (patch)

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#314]
I found a nagging typo in "ext/autosummary/templates/autosummary/module.rst" of autosummary, I also attached a patch to fix it. Following typo leads to printing classes twice, first to classes section and second time under **exceptions**, where they probably does not belong :)

```
#!text

   .. rubric:: Exceptions

   .. autosummary::
   {&#37; for item in classes &#37;}
      {{ item }}
   {&#37;- endfor &#37;}
```

Change "classes" to "exceptions", and all is fine after that.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/314
- Originally reported by: [ciantic](https://bitbucket.org/ciantic)
- Originally created at: 2010-01-11T13:49:00.523

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-01-12 09:56:02+00:00_

Thanks, fixed in <<cset a9235ebc238db493076367d605b2f5c65eba7d3a>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-22 10:14:40+00:00_

Removing milestone: 1.0 (automated comment)
